index.html

```diff
@@ -1,14 +1,168 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Mockup Home</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <section id="header">
+
+    <h1>HOME PAGE</h1>
+
+    <h5>THE PAGE FOR THE HOME</h5>
+  </section>
+
+  <hr>
+  <hr>
+<!--Section for three images and their labels, and the block quote-->
+  <section id="features">
+    <h4>TAKE A LOOK AT THIS AMAZING HOME PAGE</h4>
+    <div id="index-top-images">
+      <img src="https://www.placecage.com/200/200" />
+      <img src="https://www.placecage.com/200/200" />
+      <img src="https://www.placecage.com/200/200" />
+    </div>
+    <div id="features-img-captions">
+      <p>IMAGE ONE</p>
+      <p>IMAGE TWO</p>
+      <p>IMAGE THREE</p>
+    </div>
+    <div id="centered-button"><a href="learn-more.html"><button class="nav-buttons">TELL ME MORE</button></a></div>
+    <div id="block-quote-container">
+      <div id="block-quote">
+        <p>"I DID NOT HAVE TIME TO CREATE MY OWN THEME FOR THE SITE"</p>
+        <p>- KEVIN MAHONEY</p>
+      </div>
+    </div>
+  </section>
+<!--Section for the main articles and images-->

```

Comment:
This whole `"main"` section is great to see after hearing the questions you asked about page layout and using flex box on Friday!